packages/shared/src/components/accordion/index.tsx

```diff
@@ -7,21 +7,34 @@ import { ArrowIcon } from '../icons';
 interface AccordionProps {
   title: ReactNode;
   children: ReactNode;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
 }
 
-export function Accordion({ title, children }: AccordionProps): ReactElement {
+export function Accordion({
+  title,
+  children,
+  onClick,
+}: AccordionProps): ReactElement {
   const [isOpen, setIsOpen] = useState(false);
   const id = useId();
   const contentId = `accordion-content-${id}`;
 
+  const handleClick: MouseEventHandler<HTMLButtonElement> = (e) => {
+    if (onClick) {
+      onClick(e);
+    }
```

Comment:
```suggestion
    onClick?.(e);
```

Is also an option

Reply:
Yeah, we used to do this, but way way back, it was said that curly braces were preferred. Although I don't mind using the single-liner.